sites/all/modules/theme2_blocks_panels_menus/theme2_blocks_panels_menus.ctools_content.inc

```diff
@@ -116,7 +116,7 @@ function theme2_blocks_panels_menus_default_ctools_custom_content() {
   $content->settings = array(
     'admin_title' => '',
     'title' => 'JOIN THE MOVEMENT',
-    'body' => '<hr><a href="https://debugs.slack.com" id="apply-today">APPLY TODAY<img src="/sites/all/themes/debuga_theme2/images/apply-today-button.png"></a>',
+    'body' => '<hr><a href="/apply" id="apply-today">APPLY TODAY<img src="/sites/all/themes/debuga_theme2/images/apply-today-button.png"></a>',
```

Comment:
So that a different theme could use this as well, it is much better not to 'hardcode' the image.
Something like this would be better:
<hr><a href="/apply" id="apply-today">APPLY TODAY</a>
/* In the theme's css file: */
#id {
padding-right: (WIDTH-OF-ARROW-IMAGE)px;
background: url('images/apply-today-button.png') top right no-repeat;
}


For our purposes, this isn't an issue - but I wanted you to be aware of it.